.github/workflows/issues_and_pull_requests.yml

```diff
@@ -33,6 +33,12 @@ jobs:
 
     runs-on: "ubuntu-latest"
 
+    # See: https://github.com/actions/stale#recommended-permissions
+    permissions:
+      contents: write # only for delete-branch option
```

Comment:
As the `delete-branch` option is not used, you don't need to give this permission.

Reply:
Yeah, but it does not break anything and if we decide at any point to use that option, then it'll work 😉. Do you know which permission is required to store the cache properly?